{{cookiecutter.repo_name}}/README.rst

```diff
@@ -1,23 +1,152 @@
 README for {{cookiecutter.project_desc}}
 
+
 Testing with Docker
 -------------------
 
 This XBlock comes with a Docker test environment ready to build, based on the xblock-sdk workbench. To build and run it::
 
-        $ make dev.run
+    $ make dev.run
 
 The XBlock SDK Workbench, including this XBlock, will be available on the list of XBlocks at http://localhost:8000
 
 Translating
 -----------
 
+Internationalization (i18n) is when a program is made aware of multiple languages.
+Localization (l10n) is adapting a program to local language and cultural habits.
+
 Use the locale directory to provide internationalized strings for your XBlock project.
+For more information on how to enable translations, visit the
+`Open edX XBlock tutorial on Internationalization <http://edx.readthedocs.org/projects/xblock-tutorial/en/latest/edx_platform/edx_lms.html>`_.
+
+This cookiecutter template uses `django-statici18n <https://django-statici18n.readthedocs.io/en/latest/>`_
+to provide translations to static javascript using ``gettext``.
+
+The included Makefile contains targets for extracting, compiling and validating translatable strings.
+The general steps to provide multilingual messages for a Python program (or an XBlock) are:
+
+1. Mark translatable strings.
+2. Run i18n tools to create raw message catalogs.
+3. Create language specific translations for each message in the catalogs.
+4. Use ``gettext`` to translate strings.
+
+1. Mark translatable strings
```

Comment:
Can you also make all the sections in this file semantically hierarchical, besides using numbers?  See http://docutils.sourceforge.net/docs/user/rst/quickstart.html#sections

For instance,
```
Testing with Docker
===================
Translating
===========
1. Mark translatable strings
-----------------------------
etc.
```